src/db.rs

```diff
@@ -1123,6 +1123,17 @@ impl DB {
             );
         }
     }
+
+    pub fn set_option(&self, name: &str, value: &str) -> Result<(), Error> {
+        let cname = CString::new(name.as_bytes()).unwrap();
```

Comment:
To use `unwrap()` here it's a bad idea. 
```suggestion
         let cname = CString::new(name.as_bytes())?;
```